resources-test/src/appleMain/kotlin/dev/icerock/moko/resources/test/MockResourcesFactory.kt

```diff
@@ -6,8 +6,10 @@ package dev.icerock.moko.resources.test
 
 import dev.icerock.moko.resources.ColorResource
 import dev.icerock.moko.resources.FileResource
+import dev.icerock.moko.resources.FontResource
 import dev.icerock.moko.resources.StringResource
 
 actual fun createStringResourceMock(): StringResource = StringResource("")
 actual fun createFileResourceMock(): FileResource = FileResource("", "")
 actual fun createColorResourceMock(): ColorResource = ColorResource("")
+actual fun createFontResourceMock(): FontResource = FontResource("")
```

Comment:
Moving this to `appleMain` ended up not being necessary to build, but it makes the code a little cleaner.